util/design/lib/OtpMemImg.py

```diff
@@ -115,6 +118,9 @@ def _to_hexfile_with_ecc(data, annotation, config):
         # ECC encode
         word_bin = format(word, bin_format_str)
         word_bin = ecc_encode(config, word_bin)
+        # Pad to byte boundary and permute data if needed
```

Comment:
Nit: Should this be a "word boundary"? (Looking at the terms that go into `bit_padding`)

Reply:
Actually, both statements are correct in this case (mathematically) ;). I'll go with "word boundary" as it seems more natural in this case.